explorer/websockethandlers.go

```diff
@@ -250,15 +250,17 @@ func (exp *explorerUI) RootWebsocket(w http.ResponseWriter, r *http.Request) {
 				}
 
 				if !sig.IsValid() {
-					break loop
+					log.Errorf("invalid signal to send: %s / %d", sig.Signal.String(), int(sig.Signal))
+					continue
 				}
 
 				log.Tracef("signaling client: %p", &updateSig)
 
 				// Write block data to websocket client
 
 				webData := WebSocketMessage{
-					EventId: sig.String(),
+					// Use HubSignal's string, not HubMessage's string, which is decorated.
+					EventId: sig.Signal.String(),
```

Comment:
Also was a bug.  e.g. a `HubMessage` for a new transactions message is like `"newtxs:len=1"`, but the websocket client needs the `HubSignal`'s string (`"newtxs").